docs/execute-methods.md

```diff
@@ -1328,6 +1328,19 @@ Checks if the system on-screen keyboard is visible.
 
 `true` if the keyboard is visible
 
+### mobile: keys
+
+Send keys to the given element or to the application under test.
+This API is only supported since Xcode 15/iOS 17.
```

Comment:
Have you tested in ios 17 for iphone?

https://developer.apple.com/documentation/xctest/xcuielement/1500604-typekey?language=objc said it is available with iPadOS, so was wondering if here would be `iPadOS` instead of `iOS`.

I'll test this with real devices before approving this (I want to check the behavior), anyway

Reply:
as far as I can see XCTest does not make any difference there:

```
#if TARGET_OS_OSX || TARGET_OS_MACCATALYST || TARGET_OS_IOS

/*!
 * Hold modifier keys while the given block runs. This method pushes and pops the modifiers as global state
 * without need for reference to a particular element. Inside the block, elements can be clicked on, dragged
 * from, typed into, etc.
 */
+ (void)performWithKeyModifiers:(XCUIKeyModifierFlags)flags block:(XCT_NOESCAPE void (^)(void))block API_AVAILABLE(ios(15.0), macCatalyst(13.0));

/*!
 * Types a single key with the specified modifier flags. Although `key` is a string, it must represent
 * a single key on a physical keyboard; strings that resolve to multiple keys will raise an error at runtime.
 * In addition to literal string key representations like "a", "6", and "[", keys such as the arrow keys,
 * command, control, option, and function keys can be typed using constants defined for them in XCUIKeyboardKeys.h
 */
- (void)typeKey:(NSString *)key modifierFlags:(XCUIKeyModifierFlags)flags;

#endif // TARGET_OS_OSX || TARGET_OS_MACCATALYST || TARGET_OS_IOS
```

Reply:
In my real device, the WDA code itself worked but no input occurred on an iPhone. An iPad had inputs like `i`, space etc

Reply:
ok, I'll add a comment about it